HW_7/logger.py

```diff
@@ -0,0 +1,10 @@
+from datetime import datetime
+
+
+def log(func):
+    def wapper(*args):
+        with open('log.csv', 'a', endecodind='utf-8') as log_file:
```

Comment:
Ольга, а вы точно запускали вашу программу? Просто при запуске происходит ошибка в этой строке. Все из за того, что вы неверно написали имя аргумента
```suggestion
        with open('log.csv', 'a', encoding='utf-8') as log_file:
```